app/models/organisation.rb

```diff
@@ -8,6 +8,8 @@ class Organisation < ActiveRecord::Base
   has_many :orders
   has_many :users, through: :organisations_users
 
+  scope :with_eager_load, -> { includes([:orders]) }
```

Comment:
`with_eager_load` seems very generic

can rename to `scope :with_order, -> { includes(:orders) }`

Reply:
Yup.. 👍